packages/core/src/storage-0.8.test.ts

```diff
@@ -110,3 +110,8 @@ test('user defined value types - no layout info - bad upgrade', async t => {
     },
   });
 });
+
+test('renamed retyped - extraction', async t => {
```

Comment:
Is this test case related in any way to solidity 0.8 (since this is in `storage-0.8.test.ts`)?

Reply:
`@custom` annotations were added in Solidity 0.8.2.